plugins/node/opentelemetry-plugin-ioredis/src/utils.ts

```diff
@@ -29,8 +29,13 @@ import {
 
 const endSpan = (span: Span, err: NodeJS.ErrnoException | null | undefined) => {
   if (err) {
+    let code = CanonicalCode.UNKNOWN;
+    if (err.message.startsWith('NOSCRIPT')) {
+      code = CanonicalCode.NOT_FOUND;
+    }
+
     span.setStatus({
-      code: CanonicalCode.UNKNOWN,
+      code,
```

Comment:
What are your thoughts of putting a ternary here instead of the let if before?

Reply:
good idea. pushed an update